index.html

```diff
@@ -3,7 +3,7 @@
         <title>My website</title>
         <link rel='stylesheet' href='style.css'/>
     </head>
-    <body style="background-color:bdefec">
+    <body style>
```

Comment:
There are a few places in this file where you have a style attribute like this, and you don't need it anymore. It's not hurting anything, but I'd suggest deleting these just to clean up the code a bit.